recipes/pyvisa/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.8" %}
+{% set sha256 = "52830df5abcc293c2cd027517ee79f4f3005c7e27b8b8f0562b0bd6af8fa5e3a" %}
+
+package:
+  name: pyvisa
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/hgrecco/pyvisa/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
It seems that `pyvisa` import `pkg_resources`. You may need to add `setuptools` as a run time dependency.